app/create/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var Angular    = require('angular');
+var State      = require('./state');
+var Controller = require('./controller');
```

Comment:
nit: alphabetize constants

Reply:
I'm not following here; is it _because_ that `State` and `Controller` are capitalized that it's a nit? 👁